app/view_models/account_show.rb

```diff
@@ -43,7 +39,7 @@ def show_gpo_partial?
   end
 
   def showing_any_partials?
-    show_service_provider_continue_partial? || show_manage_personal_key_partial? ||
```

Comment:
`showing_any_partials?` is used in this template, but the `show_manage_personal_key_partial?` is not used to show anything

https://github.com/18F/identity-idp/blob/57fcf601cd22d96c445cce1a8968de4a32105805/app/views/accounts/show.html.erb#L3-L25